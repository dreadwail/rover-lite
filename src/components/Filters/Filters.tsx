import React, { FC } from 'react';

import SearchOptions from '../SearchOptions';
import SearchTypes from '../SearchTypes';

const Filters: FC = () => (
  <div className="card">
    <div className="card-body">
      <form>
        <SearchTypes />
        <SearchOptions />
      </form>
    </div>
  </div>
);

export default Filters;
