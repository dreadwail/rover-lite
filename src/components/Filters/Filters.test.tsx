import { shallow, ShallowWrapper } from 'enzyme';
import React from 'react';

import Filters from './Filters';

describe('Filters', () => {
  let rendered: ShallowWrapper;

  beforeEach(() => {
    rendered = shallow(<Filters />);
  });

  it('renders correctly', () => {
    expect(rendered).toMatchSnapshot();
  });
});
