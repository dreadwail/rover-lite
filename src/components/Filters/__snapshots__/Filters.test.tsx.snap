// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filters renders correctly 1`] = `
<div
  className="card"
>
  <div
    className="card-body"
  >
    <form>
      <ConnectFunction />
      <ConnectFunction />
    </form>
  </div>
</div>
`;
